mofacts/client/views/home/profile.html

```diff
@@ -31,88 +31,124 @@
                         </div>
-                        <div class="col-12 col-md-5 mt-auto d-flex align-items-end">
-                            <div class="row" style="position: relative; top: 55px;">
-                                <div class="col-12" style="">
-                                    <input type="text" id="practiceTDFSearch" class="form-control" placeholder="Search for module">
-                                </div>
-                                <div class="col-12" id="practiceTDFSearchResults">
-                                    {{#if enabledTdfs}}
-                                        {{#each enabledTdfs}}
-                                        <div class='col-xs-12 col-sm-12 col-md-3 col-lg-3 text-center'>
-                                            <button id="{{this.tdfid}}" class="btn btn-block btn-responsive tdfButton" name="{{this.name}}"
-                                                data-tdfid="{{this.tdfid}}"
-                                                data-lessonname="{{this.name}}"
-                                                data-currentStimuliSetId="{{this.currentStimuliSetId}}"
-                                                data-tdffilename="{{this.fileName}}"
-                                                data-ignoreOutOfGrammarResponses="{{this.ignoreOutOfGrammarResponses}}"
-                                                data-speechOutOfGrammarFeedback="{{this.speechOutOfGrammarFeedback}}"
-                                                data-isMultiTdf="{{this.isMultiTdf}}">
-                                                <!-- add disabled="{{this.isOverDue}}" to this to enable due date lockouts -->
-                                                {{this.name}}
-                                            {{#if this.dueDate}}
-                                                <br>
-                                                <small>Due: {{this.dueDate}}</small>
-                                            {{/if}}
-                                            {{#if this.enableAudioPromptAndFeedback}}
-                                                <br>
-                                                <p style="display:inline-block" title="Audio Output available for this module"><span class="fa fa-headphones"></span></p>
-                                            {{/if}}
-                                            <p style="display:inline-block">&nbsp;&nbsp;&nbsp;</p>
-                                            {{#if this.audioInputEnabled}}
-                                                <p style="display:inline-block" title="Speech Input available for this module"><i class="fa fa-microphone"></i></p>
-                                            {{/if}}
-                                        </button>
-                                        </div>
+                        <div class="col-12 text-center" id="practiceTDFSearchResultsContainer">
```

Comment:
This container covers the whole screen and prevents the user from being able to launch any TDFs

Reply:
doi, opacity instead of visibility.

Reply:
@MegaGeese it's safe now